src/Components/AffectedClustersTable/AffectedClustersTable.js

```diff
@@ -9,16 +9,17 @@ import PrimaryToolbar from '@redhat-cloud-services/frontend-components/PrimaryTo
 import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTable';
 import { TableToolbar } from '@redhat-cloud-services/frontend-components/TableToolbar';
 import { DateFormat } from '@redhat-cloud-services/frontend-components/DateFormat';
-import { Card, CardBody } from '@patternfly/react-core/dist/js/components/Card';
-import { Table } from '@patternfly/react-table/dist/js/components/Table/Table';
-import { TableBody } from '@patternfly/react-table/dist/js/components/Table/Body';
-import { TableHeader } from '@patternfly/react-table/dist/js/components/Table/Header';
-import { Bullseye } from '@patternfly/react-core/dist/js/layouts/Bullseye';
-import { Tooltip } from '@patternfly/react-core/dist/js/components/Tooltip';
 import {
+  Card,
+  CardBody,
+  Tooltip,
+  Bullseye,
   Pagination,
   PaginationVariant,
-} from '@patternfly/react-core/dist/js/components/Pagination/Pagination';
+} from '@patternfly/react-core/';
```

Comment:
```suggestion
} from '@patternfly/react-core';
```

Reply:
hey this is not the only place I found trailing slash.. please check all where it occurs and clean it please. otherwise LGTM